app/styles/base/forms.scss

```diff
@@ -296,3 +296,7 @@ input.search-input {
     border: none;
     outline: none;
 }
+
+bl-submit-btn {
```

Comment:
nah i think it works fine with the way we do forms. every form has a submit button (or 2) and then a cancel button which is just a <button />. I have yet to see an example of it adding padding where it shouldn't and if i do i will fix it. Though i could nest it under: ".form-buttons" which would narrow it down a little more.